frontend/src/views/WheresMyGene/common/store/actions.ts

```diff
@@ -3,22 +3,15 @@
  * in `./reducer.ts`.
  */
 
+import { CellTypeMetadata } from "../../components/HeatMap/utils";
+import { CellTypeSummary } from "../types";
 import { REDUCERS, State } from "./reducer";
 
-export function deleteSelectedGenesAndSelectedCellTypeIds({
-  genesToDelete,
-  cellTypeIdsToDelete,
-}: {
-  genesToDelete: string[];
-  cellTypeIdsToDelete: string[];
-}): GetActionTypeOfReducer<
+export function deleteSelectedGenesAndSelectedCellTypeIds(): GetActionTypeOfReducer<
   typeof REDUCERS["deleteSelectedGenesAndSelectedCellTypeIds"]
 > {
   return {
-    payload: {
-      cellTypeIdsToDelete,
-      genesToDelete,
-    },
+    payload: null,
```

Comment:
What was the payload, is now held in redux store, correct?

Reply:
That's right!